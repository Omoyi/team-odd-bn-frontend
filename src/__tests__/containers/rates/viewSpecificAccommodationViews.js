import React from 'react';
import { shallow } from 'enzyme';
import {
  viewSpecificAccommodationViewsMocks,
  mapStateToProspReviews
} from '../../../__mocks__/RatingMocks/BookingAccommodation';
import {
  ViewSpecificAccommodation,
  mapStateToProps,
} from '../../../views/accommodation/ViewSpecificAccommodation';

describe('Expect to view specific details information', () => {
  const wrapper = shallow(<ViewSpecificAccommodation {...viewSpecificAccommodationViewsMocks} />);
  it('Should view specific accommodation', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('Expect to view data from map state to props', () => {
    const result = mapStateToProps(mapStateToProspReviews.state);
    expect(result).toEqual({ accommodationItem: { rates: 'accommodation items follows' } });
  });
});
