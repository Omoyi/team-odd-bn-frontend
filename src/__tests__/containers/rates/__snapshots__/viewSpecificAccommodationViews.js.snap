// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to view specific details information Should view specific accommodation 1`] = `ShallowWrapper {}`;
